src/units/abilities.cpp

```diff
@@ -1297,48 +1297,48 @@ static bool overwrite_special_affects(const config& special)
 	return (apply_to == "one_side" || apply_to == "both_sides");
 }
 
-unit_ability_list attack_type::overwrite_special_checking(const std::string& ability, unit_ability_list temp_list, const unit_ability_list& abil_list, const std::string& filter_self) const
+unit_ability_list attack_type::overwrite_special_checking(const std::string& ability, const unit_ability_list& temp_list, const unit_ability_list& abil_list, const std::string& filter_self, bool is_special) const
 {
 	bool overwrite_self = false;
+	bool overwrite_oppponent = false;
```

Comment:
Just noticed: there should only be two `p`s in `opponent`.

Reply:
corrected

Reply:
Sorry, you missed a couple. `src/units/abilities.cpp:1314:187: error: ‘overwrite_oppponent’`, and another on line 1315.

Reply:
fixed